pages/api/google/export/roster.js

```diff
@@ -6,10 +6,13 @@ module.exports = async (req, res) => {
     if (req.method === 'POST') {
         
         // Get the Access Token from the request headers
-        const token = cookie.parse(req.headers.cookie).token;
-        const authStatus = await data.auth.checkAnyUser(`Bearer ${token}`, res);
+        //const token = cookie.parse(req.headers.cookie).token;
+        //console.log(token)
+        //const authStatus = await data.auth.checkAnyUser(`Bearer ${token}`, res);
+
+        // test test
                             
-        if (authStatus) {
+        if (true) {
```

Comment:
Please revert these changes :) .

Sorry, my fault for not preparing the DEV environment appropriately here.